src/preact/base-element.js

```diff
@@ -725,6 +729,26 @@ function collectProps(Ctor, element, ref, defaultProps) {
       devAssert(def.attrs);
       value = def.parseAttrs(element);
     }
+    if (def.attrPrefix) {
+      const currObj = {};
+      const attrs = element.attributes;
+      for (let i = 0; i < attrs.length; i++) {
+        const attrib = attrs[i];
+        if (
+          attrib.name.startsWith(def.attrPrefix) &&
```

Comment:
Super nit, but could you please move the `startsWith() && !==` code into a top-level function in this file and call it something like `matchesAttrPrefix`? It's used here and in the mutation filtering code.